app/v1/services/RetrieveChargeHistoryService.scala

```diff
@@ -50,18 +50,18 @@ class RetrieveChargeHistoryService @Inject() (connector: RetrieveChargeHistoryCo
 
```

Comment:
Is the for comprehension on line 44 necessary?

Reply:
This question applies to almost all the services in v1/services

Reply:
It's outside the scope of the PR again, but I agree the for-comp there isn't needed :slightly_smiling_face: